ledger/complete/wal/wal.go

```diff
@@ -37,14 +32,12 @@ func NewDiskWAL(logger zerolog.Logger, reg prometheus.Registerer, metrics module
 		return nil, err
 	}
 	return &DiskWAL{
```

Comment:
I don't think `metrics` is used any more in `NewDiskWAL()`.  Maybe we can remove it, or replace it with `_`?

If we choose to remove `metrics`, maybe we can also remove `WALMetrics` because it's an empty interface and it isn't used anywhere else.

Reply:
yeah, we could remove the metrics and the WALMetrics interface, but it breaks lots of tests. Fixing them will have to introduce lots of changes.

I'd like to keep the metrics as is, in case we need some metrics for WAL, then we don't have to add the metrics back and go through test cases fixing again.